### DISCUSSION TITLE
How do I render a bokeh graph from my jup notebook?

### DISCUSSION THREAD

**Discussion by EduardoLZevallos** [quarto-dev/quarto-cli#7483] [Q&A]
### Description

I have a jupyter notebook that I am rendering that has a cell that shows a bokeh graph.  However when I render the notebook the graph doesn't appear.  How do I fix this issue? I am using show(p) to render the bokeh graph

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````
